src/tweet.js

```diff
@@ -88,10 +88,44 @@ function tweetNow(status){
     });
 }
 
+/**
+ *
+ * @description Sends dms to all followers
+ */
```

Comment:
@Crammers Did you check whether it's working or not?
Please set up your own Twitter Application, get API keys and test it.

Reply:
Yes and No,
Yes the function works and I've sent DM's on my own account but testing with the Twitter packageI could not get past an authentication error. I could not even do a basic get or post (using one of the functions in your code and one of my own) so I'm not sure why.

Seems like there's no updates to the Twitter library for 2 years and a number of issues have been raised with similar problems.

The Twit package works the same as this Twitter one and I've no issue connecting or sending tweets/DM's.

Reply:
Can you please confirm? 
If the `npm` module we're using outdated now, we can use another one i.e. Twit.

Reply:
I believe it is. If you want let me test all the current code work the same with Twit and if so I'd recommend changing to it.

Reply:
Yes, please go ahead. Update the code and check all the flows.

Reply:
The only difference I can see between Twit and Twitter is the **access_token_key** is a slightly different name. In Twit its just **access_token**.  I'll not be able to update this tonight but will do it first thing in the morning.

Reply:
Cool:)
No problem. It's Hacktoberfest. We've time till the end of October. 😄

Reply:
Ive updated the PR, I was able to update the Twitter package to Twit without issue. 
I haven't added the function to be called from any of the other function, twitter can be a bit strict with [rate limiting](https://developer.twitter.com/en/docs/basics/rate-limiting) so it would be easily exceeded if there were just a handful of followers. So I wasn't sure what would be the best method of implementation to avoid blocking?